pkg/logging/config_test.go

```diff
@@ -50,9 +50,7 @@ func TestNewConfigNoEntry(t *testing.T) {
 	if diff := cmp.Diff(cm, c); diff != "" {
 		t.Fatalf("Config mismatch: diff(-want,+got):\n%s", diff)
 	}
-	if got := c.LoggingConfig; got == "" {
-		t.Error("LoggingConfig = empty, want not empty")
-	}
+	// TODO: assert that c.LoggingConfig is empty.
```

Comment:
Why there's TODO here?

Reply:
https://github.com/knative/pkg/pull/1773 will change the behavior such that an empty logging config will get parsed to an empty LoggingConfig. After merging 1773 we can an add an assertion of that behavior, i.e. the opposite of the old assertion.